server/utils/features.js

```diff
@@ -7,7 +7,7 @@ const LIVERPOOL_PRE_PILOT_USERS = ['TaylorColinoNPS', 'debbieleenps', 'emmacaddi
 
 const PLYMOUTH_MAGS_COURT_CODE = 'B50KH'
```

Comment:
Interesting that this code is already here

Reply:
I didn't notice that.... made a change